src/le.js

```diff
@@ -1,6 +1,6 @@
 /**
- * @license Copyright 2015 Logentries.
- * Please view license at https://raw.github.com/logentries/le_js/master/LICENSE
+ * @license Copyright 2019 Rapid7.
+ * Please view license at https://raw.github.com/rapid7/r7insight_js/master/LICENSE
```

Comment:
Is this deliberately changed from `le_js` to `r7insight_js` ?

Reply:
Yes, it serves the License in the repo. The public one will be modified when it will be merged